action.yml

```diff
@@ -28,16 +32,44 @@ runs:
         key: dune-cache-${{ runner.os }}-${{ runner.arch }}
     - name: Resolve dependencies
       shell: bash
+      env:
+        SETUPDUNEDIR: ${{ inputs.directory }}
+      run: |
+        cd "$SETUPDUNEDIR" && dune pkg lock
+      if: inputs.automagic == 'true'
+    - name: Install depexts
+      shell: bash
+      env:
+        SETUPDUNEDIR: ${{ inputs.directory }}
+        OS: ${{ runner.os }}
       run: |
-        dune pkg lock
+        DEPEXTS="$(cd "$SETUPDUNEDIR" >/dev/null && dune show depexts 2>&1)"
+        case "$OS,$DEPEXTS" in
```

Comment:
Do you know what is the semantic of bash `case` when there is no match? (Silently do nothing?).

For comparison, I had this in the other version:

```sh
  echo "[install-depexts] Depext installation is not supported on this OS: ${RUNNER_OS}"
  exit 1
```

Reply:
Yes, silent do nothing. Note that the dune installation would have broken before on Windows, so I didn’t feel it useful to cover that case.